css/style.css

```diff
@@ -0,0 +1,112 @@
+body {
+  font-family: sans-serif;
+  font-size: 18px;
+  line-height: 1.5;
+  margin: 30px;
+  border-bottom: 5px solid #222222;
+  border-top: 5px solid #222222;
+  padding: 50px;
+  background: grey;
+  
+}
+
+
+h1 {
+  color: white;
+  font-family: 'Rock Salt', cursive;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+
+
+
+
+h2 {
+  background-color: white;
+  font-family: 'Monoton', cursive;
+}
+
+
+
+
+
+
+h3 {
+  color: white;
+  font-size: 24px;
+
+}
+
+h4 {
+  font-family: 'Bungee Inline', cursive;
+
+}
+
+
+p {
+  font-family: 'Cherry Swash', cursive;
+
+
+}
+
+
+hr {
+  border: none;
+  border-top: 1px solid #222222;
```

Comment:
Good opportunity for using the hex color shorthand (http://www.websiteoptimization.com/speed/tweak/hex/). 🔨